react/ui-components/src/hoc/FieldV1.js

```
@@ -329,23 +342,52 @@ const FieldV1 = ({
+            variant={
+              variant
+                ? variant
+                : errors?.[populators?.name]
+                  ? "digit-field-error"
+                  : ""
+            }
+          />
+        );
+      case "boundary":
+        // Use value from form (defaultValues) if available, otherwise fall back to populators.preSelected
+        const preSelectedValue = value || populators.preSelected;
```

Biome (2.3.13)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)